chapters/exampleComplex.json

```diff
@@ -22,14 +22,21 @@
       "startTime": 410,
       "title": "Namespace",
       "img": "https://example.com/images/namepsace_example.jpg",
-      "url": "https://github.com/Podcastindex-org/podcast-namespace"
+      "url": "https://github.com/Podcastindex-org/podcast-namespace",
     },
     {
       "startTime": 3990,
       "title": "Just Break Up",
       "img": "https://example.com/images/justbreakuppod.png",
       "url": "https://twitter.com/justbreakuppod"
     },
+     {
+      "startTime": 4200,
+      "title": "Played song by artist",
+      "img": "https://i.discogs.com/R-249504-1334592212.jpeg?bucket=discogs-images&fit=contain&format=auto&height=600&quality=90&width=600&signature=wMI9I0mHkbVxkkryQrN1JkkzhwFsrereuom9Lmfa92w%3D",
+      "url": "https://www.discogs.com/Rick-Astley-Never-Gonna-Give-You-Up/master/96559"
+      "valueOverride": "<podcast:value type=\"lightning\" method=\"keysend\" suggested=\"0.00000005000\"><podcast:valueRecipient name=\"Performing Artist\" type=\"node\" address=\"03c457fafbc8b91b462ef0b8f61d4fd96577a4b58c18b50e59621fd0f41a8ae1a4\" split=\"100\"/></podcast:value>"
```

Comment:
Any reason to include the explicit "Override" in the name? 

I think simply calling it `value` would be clear and in line with the `podcast:value` block definition, which states:

> The <podcast:value> tag designates the cryptocurrency or payment layer that will be used, the transport method for transacting the payments, and a suggested amount denominated in the given cryptocurrency.
> This element can exist at either the <channel> or <item> level.

Reply:
I agree with this.